src/textual/app.py

```diff
@@ -2375,18 +2375,22 @@ async def action_pop_screen(self) -> None:
         self.pop_screen()
 
     async def action_back(self) -> None:
+        """Go back to the previous screen (pop the current screen)."""
         try:
             self.pop_screen()
         except ScreenStackError:
             pass
 
     async def action_add_class_(self, selector: str, class_name: str) -> None:
+        """Add a CSS class on the selected widget."""
         self.screen.query(selector).add_class(class_name)
 
     async def action_remove_class_(self, selector: str, class_name: str) -> None:
+        """Remove a CSS class on the selected widget."""
         self.screen.query(selector).remove_class(class_name)
 
     async def action_toggle_class(self, selector: str, class_name: str) -> None:
+        """Toggle a CSS class on the selected widget."""
```

Comment:
Should probably add the `Args` while here.

Reply:
Sorry, I meant to make this PR a draft.

Reply:
But I'm sure this will be a perfect PR and can be pre-approved. :-P